### ISSUE TITLE
beta 23 bug?

### ISSUE THREAD

**Issue by musikboden** [paulrosen/abcjs#567]
Running this song in "editor-synth" will cause an error

X:1
T:My funny valentine
T:kompmodell
C:Rodgers/Hart
%%score { 1 | 2 }
L:1/4
Q:1/4=100
M:4/4
I:linebreak $
K:Eb
V:1 treble nm="Piano"
V:2 bass 
V:1
 "Cm"[GEC]2 [GEC]2 | "G/B"[GD=B,]2 [GDB,]2 | "Cm/Bb"[GEC]2 [GEC]2 | "Am7b5"[GEC]2 [GEC]2 | %4
 "Abmaj7"[GEC]2 [GEC]2 | "Fm7"[AEC]2 [AEC]2 | "Dm7b5"[AFC]2 [AFC]2 | "G7"[GF=B,]2 [GFB,]2 | %8
 "Cm"[GEC]2 [GEC]2 | "G/B"[GD=B,]2 [GDB,]2 | "Cm/Bb"[GEC]2 [GEC]2 | "Am7b5"[GEC]2 [GEC]2 | %12
 "Abmaj7"[GEC]2 [GEC]2 | "Fm7"[AEC]2 [AEC]2 | "Abm6"[AF=B,]2 [AFB,]2 | "Bb7"[AFD]2 [AFD]2 || %16
 "Ebmaj7"[GDB,]2 "Fm9"[GEA,]2 | "Gm"[GDB,]2 "Fm9"[GEA,]2 | "Ebmaj7"[GDB,]2 "Fm9"[GEA,]2 | "Gm"[GDB,]2 "Fm9"[GEA,]2 | %20
 "Ebmaj7"[GDB,]2 "G7"[GF=B,]2 | "Cm"[GEC]2 "Eb7/Bb"[GE_D]2 | "Abmaj7"[GEC]2 [GEC]2 | "Dm7b5"[AFC]2 "G7"[GF=B,]2 || %24
 "Cm"[GEC]2 [GEC]2 | "G/B"[GD=B,]2 [GDB,]2 | "Cm/Bb"[GEC]2 [GEC]2 | "Am7b5"[GEC]2 [GEC]2 | %28
 "Abmaj7"[GEC]2 [GEC]2 | "Dm7b5"[AFC]2 "G7"[GF=B,]2 | "Cm"[GEC]2 [GEC]2 | "Eb7/Bb"[GE_D]2 "A7#11"[GED]2 | %32
 "Abmaj7"[GEC]2 [GEC]2 | "Fm"[FCA,]2 "Bb7"[FDA,]2 | "Eb"[EB,G,]4- | [EB,G,]4 |] %36
V:2
 C,4 | =B,,3 B,, | _B,,4 | =A,,3 A,, | %4
 _A,,4 | F,,4 | D,4 | G,,3 G,, | %8
 C,4 | =B,,3 B,, | _B,,4 | =A,,3 A,, | %12
 _A,,4 | F,,4 | A,,4 | _B,,3 B,, || %16
 E,,2 F,,2 | G,,2 F,,2 | E,,2 F,,2 | G,,2 F,,2 | %20
 E,,2 G,,2 | C,2 _B,, =A,, | _A,,4 | D,2 G,,2 || %24
 C,4 | =B,,3 B,, | _B,,4 | =A,,3 A,, | %28
 _A,,4 | D,2 G,,2 | C,4 | B,,2 =A,,2 | %32
 _A,,4 | F,,2 B,,2 | E,,4- | E,,4 |] %36

error:

abcjs-basic.js:1737 Uncaught (in promise) TypeError: Cannot read property 'left' of undefined
    at skipTies (abcjs-basic.js:1737)
    at addEndPoints (abcjs-basic.js:1749)
    at Object.Tune.setupEvents (abcjs-basic.js:1718)
    at Object.Tune.setTiming (abcjs-basic.js:1812)
    at TimingCallbacks.self.replaceTarget (abcjs-basic.js:346)
    at new TimingCallbacks (abcjs-basic.js:356)
    at abcjs-basic.js:16189

**Comment by musikboden** [comment]
This song will produce same errors as above

X:1
T:Molltrollet
T:"Lilla pianoraketen"
C:I Kullberg, H Råberg, M Strömblad
%%score { 1 | 2 }
%%printtempo 0
Q:1/4 = 120
L:1/4
M:3/4
R:D-läge
I:linebreak $
K:Dm
V:1 treble nm="Piano"
V:2 bass 
V:1
%%MIDI gchordon
%%text Förspel
!1!D F A | [!1!D !3!F !5!A]3 | [D F A]-3 | [D F A]3 ||
D D D | F F F | A A A | F2 F | 
w: Högt upp på | berg-et det | sit-ter ett | troll, och |
A A A | F F F | D3- | D3 ||
w: sjung-er en | vi-sa i | moll. |
%%text Efterspel
D F A | [D F A]3 | [D F A]-3 | [D F A]3 |]
V:2
"_5"D, F, A, | "_1;3;5"[D,F,A,]3 | [D,F,A,]-3 | [D,F,A,]3 ||
D,3 | F,3 | A,3 | F,3 | 
A,3 | F,3 | D,3- | D,3 ||
D, F, A, | [D,F,A,]3 | [D,F,A,]-3 | [D,F,A,]3 |]

**Comment by musikboden** [comment]
Here I have some updates:

This song:

X:1
T:Bjällerklang
T:"Lilla pianoraketen"
C:S Pierpont, Erik Sandström, Gösta Westerberg
%%score { 1 | 2 }
%%printtempo 0
Q:1/4 = 70
L:1/8
M:4/4
R:D-läge
I:linebreak $
K:D
V:1 treble nm="Piano"
V:2 bass 
V:1
!3!FFF2 FFF2| F!5!A !1!D!2!E !3!F4 | !4!GG G!5!A !4!G!3!F FF |
w: Bjäl-ler-klang, bjäl-ler-klang | hör dess ding-li-dång | flin-gor som nu virv-lar om i |
w: Följ oss ut, följ oss ut | Blac-ken tra-var på | i hans spår vår slä-de går, där |
|1 FE EF E4 :|2 AA GE D4 |]
w: mun-ter vin-ter-sång | hö-ga fu-ror stå. |]
V:2
"_5"D,4 "_1"A,4 | D,4 A,4 | "_4"E,4 D,4 |
|1 E,4 A,4 :|2 E,4 D,4 |]

will produce this errors
Uncaught (in promise) TypeError: Cannot set property 'endX' of undefined
    at Object.Tune.setupEvents (abcjs-basic.js:1704)
    at Object.Tune.setTiming (abcjs-basic.js:1812)
    at TimingCallbacks.self.replaceTarget (abcjs-basic.js:346)
    at new TimingCallbacks (abcjs-basic.js:356)
    at abcjs-basic.js:16189
Tune.setupEvents @ abcjs-basic.js:1704
Tune.setTiming @ abcjs-basic.js:1812
TimingCallbacks.self.replaceTarget @ abcjs-basic.js:346
TimingCallbacks @ abcjs-basic.js:356
(anonymous) @ abcjs-basic.js:16189
Promise.then (async)
doNext @ abcjs-basic.js:14875
acResumerMiddleWare @ abcjs-basic.js:14869
(anonymous) @ abcjs-basic.js:14898

but if I remove the second systems first bar line with the repeat number "1", and move the "1" to the end of system 1, like this:

X:1
T:Bjällerklang
T:"Lilla pianoraketen"
C:S Pierpont, Erik Sandström, Gösta Westerberg
%%score { 1 | 2 }
%%printtempo 0
Q:1/4 = 70
L:1/8
M:4/4
R:D-läge
I:linebreak $
K:D
V:1 treble nm="Piano"
V:2 bass 
V:1
!3!FFF2 FFF2| F!5!A !1!D!2!E !3!F4 | !4!GG G!5!A !4!G!3!F FF |1
w: Bjäl-ler-klang, bjäl-ler-klang | hör dess ding-li-dång | flin-gor som nu virv-lar om i |
w: Följ oss ut, följ oss ut | Blac-ken tra-var på | i hans spår vår slä-de går, där |
 FE EF E4 :|2 AA GE D4 |]
w: mun-ter vin-ter-sång | hö-ga fu-ror stå. |]
V:2
"_5"D,4 "_1"A,4 | D,4 A,4 | "_4"E,4 D,4 |1
 E,4 A,4 :|2 E,4 D,4 |]

the error is gone

**Comment by musikboden** [comment]
And with this song (different errors)

X:1
T:Molltrollet
T:"Lilla pianoraketen"
C:I Kullberg, H Råberg, M Strömblad
%%score { 1 | 2 }
%%printtempo 0
Q:1/4 = 120
L:1/4
M:3/4
R:D-läge
I:linebreak $
K:Dm
V:1 treble nm="Piano"
V:2 bass 
V:1
%%MIDI gchordon
%%text Förspel
!1!D F A | [!1!D !3!F !5!A]3 | [D F A]-3 | [D F A]3 ||
D D D | F F F | A A A | F2 F | 
w: Högt upp på | berg-et det | sit-ter ett | troll, och |
A A A | F F F | D3- | D3 ||
w: sjung-er en | vi-sa i | moll. |
%%text Efterspel
D F A | [D F A]3 | [D F A]-3 | [D F A]3 |]
V:2
"_5"D, F, A, | "_1;3;5"[D,F,A,]3 | [D,F,A,]-3 | [D,F,A,]3 ||
D,3 | F,3 | A,3 | F,3 | 
A,3 | F,3 | D,3- | D,3 ||
D, F, A, | [D,F,A,]3 | [D,F,A,]-3 | [D,F,A,]3 |]

will produce this errors:

abcjs-basic.js:1737 Uncaught (in promise) TypeError: Cannot read property 'left' of undefined
    at skipTies (abcjs-basic.js:1737)
    at addEndPoints (abcjs-basic.js:1749)
    at Object.Tune.setupEvents (abcjs-basic.js:1718)
    at Object.Tune.setTiming (abcjs-basic.js:1812)
    at TimingCallbacks.self.replaceTarget (abcjs-basic.js:346)
    at new TimingCallbacks (abcjs-basic.js:356)
    at abcjs-basic.js:16189
skipTies @ abcjs-basic.js:1737
addEndPoints @ abcjs-basic.js:1749
Tune.setupEvents @ abcjs-basic.js:1718
Tune.setTiming @ abcjs-basic.js:1812
TimingCallbacks.self.replaceTarget @ abcjs-basic.js:346
TimingCallbacks @ abcjs-basic.js:356
(anonymous) @ abcjs-basic.js:16189
Promise.then (async)
doNext @ abcjs-basic.js:14875
(anonymous) @ abcjs-basic.js:14865
Promise.then (async)
acResumerMiddleWare @ abcjs-basic.js:14859
(anonymous) @ abcjs-basic.js:14898

but if I remove the tie on the 2 last bars in the bass clef, the error disappears.

like this:

X:1
T:Molltrollet
T:"Lilla pianoraketen"
C:I Kullberg, H Råberg, M Strömblad
%%score { 1 | 2 }
%%printtempo 0
Q:1/4 = 120
L:1/4
M:3/4
R:D-läge
I:linebreak $
K:Dm
V:1 treble nm="Piano"
V:2 bass 
V:1
%%MIDI gchordon
%%text Förspel
!1!D F A | [!1!D !3!F !5!A]3 | [D F A]-3 | [D F A]3 ||
D D D | F F F | A A A | F2 F | 
w: Högt upp på | berg-et det | sit-ter ett | troll, och |
A A A | F F F | D3- | D3 ||
w: sjung-er en | vi-sa i | moll. |
%%text Efterspel
D F A | [D F A]3 | [D F A]-3 | [D F A]3 |]
V:2
"_5"D, F, A, | "_1;3;5"[D,F,A,]3 | [D,F,A,]-3 | [D,F,A,]3 ||
D,3 | F,3 | A,3 | F,3 | 
A,3 | F,3 | D,3- | D,3 ||
D, F, A, | [D,F,A,]3 | [D,F,A,]-3 | [D,F,A,]3 |]

No errors

**Comment by musikboden** [comment]
I used the "editor-synth" example, beta 23
and Chrome Version 87.0.4280.67 (Officiell version) (x86_64)

**Comment by paulrosen** [comment]
I'm not seeing that problem. Odd. I do see where it is crashing and I can protect against that, but I really want to know how bad data got in and I need to reproduce for that.

**Comment by paulrosen** [comment]
This looks like two different bugs. Here is a simple case showing one of them:
```
X:1
T:repeat-at-start-of-line-crash
K:C
E8|
|1 D8 :|2 C8
```

**Comment by paulrosen** [comment]
And here's a simple case showing the other one:
```
X:1 
T:skip-ties-crash
M:4/4
K:C
E8- | E8 |]
```

I'll get a new beta out on Monday.

**Comment by musikboden** [comment]
Yes, you're right, 2 different bugs.
Great!

**Comment by paulrosen** [comment]
Fixed in beta 24

**Comment by musikboden** [comment]
Still same kind of problems

this song:

X:1
T:Comptine d'un autre ete
C:Yann Tiersen
%%score { RH | ( LH1 LH2 ) }
%%printtempo 0
L:1/16
Q:1/4=90
M:4/4
I:linebreak $
K:G
V:RH clef=treble nm="Piano"
V:LH1 clef=bass stem=down
V:LH2 clef=bass stem=up
V:RH
 z16 | z16 | z16 | z16 | %4
 |: z2 (GF G2)(Bc B8) | z2 (FG F2)(GA G8) | z2 (FE F2)(Bc B8) | z2 (FE F4- F8) :| %8
 (e6 B2- B8) | (d6 B2- B8) | (f6 B2- B8) | (f6 A2- A8) | %12
 ([gB]6 [eG]2- [eG]8) | ([gB]6 [dG]2- [dG]8) | ([fB]6 [dF]2- [dF]8) | ([fA]6 [dF]2- [dF]8) || %16
 (!>!Beb)(!>!B eb)(!>!Be b)(!>!Beb) (!>!Be)(!>!ce) | (!>!Bdb)(!>!B db)(!>!Bd b)(!>!Bdb) (!>!Bd)(!>!Ad) | (!>!FBf)(!>!F Bf)(!>!FB f)(!>!FBf) (!>!FB)(!>!GB) | 
 (!>!Ada)(!>!A da)(!>!Ad a)(!>!Ada) (!>!Ad)(!>!Gd) | (!>!Beb)(!>!B eb)(!>!Be b)(!>!Beb) (!>!Be)(!>!ce) | (!>!Bdb)(!>!B db)(!>!Bd b)(!>!Bdb) (!>!Bd)(!>!Ad) | 
 (!>!FBf)(!>!F Bf)(!>!FB f)(!>!FBf) (!>!FB)(!>!GB) | (!>!Ada)(!>!A da)(!>!Ad a)(!>!Ada) (!>!Ada2) | %24
 [V:RH clef=treble+8 transpose=12]!8va(! z2 (GF G2)(Bc B8) | z2 (FG F2)(GA G8) | z2 (FE F2)(Bc B8) | z2 (FE F4- F8) | %28
 (e6 B2- B8) | (d6 B2- B8) | (f6 B2- B8) | (f6 A2- A8) | %32
 ([gB]6 [eG]2- [eG]8) | ([gB]6 [dG]2- [dG]8) | ([fB]6 [dF]2- [dF]8) | ([fA]6 [dF]2- [dF]8) | %36
 (!>!Beb)(!>!B eb)(!>!Be b)(!>!Beb) (!>!Be)(!>!ce) | (!>!Bdb)(!>!B db)(!>!Bd b)(!>!Bdb) (!>!Bd)(!>!Ad) | (!>!FBf)(!>!F Bf)(!>!FB f)(!>!FBf) (!>!FB)(!>!GB) | 
(!>!Ada)(!>!A da)(!>!Ad a)(!>!Ada) (!>!Ad)(!>!Gd) | (!>!Beb)(!>!B eb)(!>!Be b)(!>!Beb) (!>!Be)(!>!ce) | (!>!Bdb)(!>!B db)(!>!Bd b)(!>!Bdb) (!>!Bd)(!>!Ad) | 
(!>!FBf)(!>!F Bf)(!>!FB f)(!>!FBf) (!>!FB)(!>!GB) | (!>!Ada)(!>!A da)"^rit."(!>!Ad a)(!>!Ada) (!>!Ad)(ag) | !8va)! [eBG]16 |] %45
V:LH1
 !p!E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | %4
|: E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 :| %8
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | %12
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 || %16
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | 
D,4 F,4 D,4 F,4 | E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | 
D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | %24
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | %28
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | %32
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | %36
 E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | D,4 F,4 D,4 F,4 | 
 D,4 F,4 D,4 F,4 | E,4 G,4 E,4 G,4 | D,4 G,4 D,4 G,4 | 
 D,4 F,4 D,4 F,4 | D,4 F,4 D,4 F,4 | [EB,E,]16 |] %45
V:LH2
 !p!E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | %4
 |: E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 :| %8
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | %12
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 || %16
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | 
 D2A,2D2A,2 D2A,2D2A,2 | E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | 
 D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | %24
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | %28
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | %32
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | %36
 E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | D2B,2D2B,2 D2B,2D2B,2 | 
 D2A,2D2A,2 D2A,2D2A,2 | E2B,2E2B,2 E2B,2E2B,2 | D2B,2D2B,2 D2B,2D2B,2 | 
 D2B,2D2B,2 D2B,2D2B,2 | D2A,2D2A,2 D2A,2D2A,2 | x16 |] %45

produces this errors:
Uncaught (in promise) TypeError: Cannot set property 'endX' of undefined
    at Object.Tune.setupEvents (abcjs-basic.js:1679)
    at Object.Tune.setTiming (abcjs-basic.js:1813)
    at TimingCallbacks.self.replaceTarget (abcjs-basic.js:346)
    at new TimingCallbacks (abcjs-basic.js:356)
    at abcjs-basic.js:16212
Tune.setupEvents @ abcjs-basic.js:1679
Tune.setTiming @ abcjs-basic.js:1813
TimingCallbacks.self.replaceTarget @ abcjs-basic.js:346
TimingCallbacks @ abcjs-basic.js:356
(anonymous) @ abcjs-basic.js:16212
Promise.then (async)
doNext @ abcjs-basic.js:14896
(anonymous) @ abcjs-basic.js:14886
Promise.then (async)
acResumerMiddleWare @ abcjs-basic.js:14880
(anonymous) @ abcjs-basic.js:14919

**Comment by musikboden** [comment]
Same with the song:

X:1
T:Soweto is where it's at
C:Abdullah Ibrahim
%%score { 1 | 2 }
L:1/8
Q:1/4=100
M:4/4
I:linebreak $
K:G
V:1 treble nm="Piano"
V:2 bass 
V:1
 [dBF] |: [dBF][cAE][BGD][cAE]- [cAE][BGD][AEC][BGD]- | [BGD][AEC][GDB,][AEC] [AEC][GDB,][FDA,][GDB,]- | [GDB,]4 [GEC]3 [GDC]- |
 [GDC]6 z [dBF] | [dBF][cAE][BGD][cAE]- [cAE][BGD][AEC][BGD]- | [BGD][AEC][GDB,][AEC] [AEC][GDB,][FDA,][GDB,]- |
 [GDB,]4 [GEC]3 [GDC]- |1 [GDC]6 z [dBF] :|2 [GDC]6 z2 |: [cAE] [BGD]2 [AEC]2 [GEB,]2 [GEB,]- |
 [GEB,]3 [FDA,]- [FDA,]4 :| [cAE] [BGD]2 [AEC]2 [GEB,]2 [dAG]- | [dAG]6 z [dBF] |
 [dBF][ecG][dBF][cAE] [BGD] [AEC]3 |]
V:2
 z |: G,,3 G,, C,2 C,D,- | D,3 D, D,2 D,2 | G,,3 G,, C,2 C,D,- |
 D,3 D, D,2 D,2 | G,,3 G,, C,2 C,D,- | D,3 D, D,2 D,2 |
 G,,3 G,, C,2 C,D,- |1 D,3 D, D,2 D,2 :|2 D,3 D, D,2 D,2 |: A,, B,,2 C,2 ^C,2 D,- |
 D,3 D, D,2 D,2 :| A,, B,,2 C,2 ^C,2 D,- | D,3 D, D,3 D, |
 D,3 D, D,2 D,2 |] 

abcjs-basic.js:1679 Uncaught (in promise) TypeError: Cannot set property 'endX' of undefined
    at Object.Tune.setupEvents (abcjs-basic.js:1679)
    at Object.Tune.setTiming (abcjs-basic.js:1813)
    at TimingCallbacks.self.replaceTarget (abcjs-basic.js:346)
    at new TimingCallbacks (abcjs-basic.js:356)
    at abcjs-basic.js:16212
Tune.setupEvents @ abcjs-basic.js:1679
Tune.setTiming @ abcjs-basic.js:1813
TimingCallbacks.self.replaceTarget @ abcjs-basic.js:346
TimingCallbacks @ abcjs-basic.js:356
(anonymous) @ abcjs-basic.js:16212
Promise.then (async)
doNext @ abcjs-basic.js:14896
acResumerMiddleWare @ abcjs-basic.js:14890
(anonymous) @ abcjs-basic.js:14919

**Comment by musikboden** [comment]
Let me know if you need more examples.

**Comment by paulrosen** [comment]
Ok, can reproduce.

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.25

**Comment by musikboden** [comment]
Thank you, now it works fine 🙏
